### ISSUE TITLE
Sanitize Readme Clarification

### ISSUE THREAD

**Issue by kmccullough** [fraywing/textAngular#1366]
The readme has the following note: "NOTE: Our textAngular-sanitize.js and angular.js's angular-sanitize.js are the SAME file, you must include one or the other but not both. We highly recommend using textAngular-sanitize.js as it loosens some parts of the sanitizer that are far too strict for our uses and adds some more features we need."

This seems to me a little ambiguous. Are they the "SAME" or are they different? I reverted my angular-sanitize to the version included in textAngular and did a comparison, and there does appear to be differences. Furthermore, "we highly recommend using textAngular-sanitize.js as it loosens some parts of the sanitizer" seems to  imply that they are different.

I guess my issue here is that the word "SAME" is capitalized as if to more adamantly assert that they are the same, and yet they are different and the next sentence seems to confirm that they are different.

If they are not the same,  perhaps we shouldn't say that they are the "SAME".

If they are slightly different, then it would be more helpful to specify that "they are the same,  except..." and then give an example or two (or link to appropriate documentation) of what we'd be missing out on if we didn't use textAngular-sanitize.js.

**Comment by JoelParke** [comment]
I agree that the language is not clear -- and needs to be updated.  
Can you provide a short paragraph that you believe would be helpful and then I can update it.
 -- the intent is let the user know that the textAngular-sanitize file serves the same purpose as the angular-sanitize file, but the one for textAngular allows through more html that is needed for TA to function fully. 

There is also a PR that worked through attempting to re-align the differences between the textAngular-sanitize.js and the angular-sanitize.js file.  That would make the comparison much clearer.
